sdk/trace/sampling.go

```diff
@@ -47,12 +47,12 @@ const (
 	// Drop will not record the span and all attributes/events will be dropped.
 	Drop SamplingDecision = iota
 
-	// Record indicates the span's `IsRecording() == true`, but `Sampled` flag
-	// *must not* be set.
+	// RecordOnly indicates the span's IsRecording method returns true, but trace.FlagsSampled flag
```

Comment:
```suggestion
	// RecordOnly indicates the span's IsRecording method returns true, but [trace.FlagsSampled] flag
```

https://tip.golang.org/doc/comment#doclinks

Reply:
Doc links are not rendered for field comments: https://github.com/open-telemetry/opentelemetry-go/pull/6257#discussion_r1943762141